umbraco.sln

```diff
@@ -34,29 +34,6 @@ Project("{E24C65DC-7377-472B-9ABA-BC803B73C61A}") = "Umbraco.Web.UI.Client", "ht
 		StartServerOnDebug = "false"
 	EndProjectSection
 EndProject
-Project("{E24C65DC-7377-472B-9ABA-BC803B73C61A}") = "Umbraco.Web.UI.New.Client", "http://localhost:3961", "{17E2067E-9D0F-4891-B6BA-CD828886B968}"
```

Comment:
Looks like some merge issues here, the `Umbraco.Web.UI.New.Client` project is missing from the solution

Reply:
Or is this intentional and I'm just missing something? :)

Reply:
Yeah, I saw that but my IDE suggested it twice to me, even though I discarded the changes. But let me have a look again 😉